tests/flow/test_persistency.py

```diff
@@ -203,3 +204,29 @@ def test04_repeated_edges(self):
             actual_result = graph.query(q)
             self.env.assertEquals(actual_result.result_set, expected_result)
 
+        # Test decoding of graph meta keys in different sequences.
```

Comment:
In a replica-of configuration,
Where the replica has both master and slave shards, please make sure the AUX field event is replicated to the slave shard.

Reply:
Please we are able to load when using AOF and the AOF file contains RESTORE commands.